__tests__/endpoints.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require('supertest');
+const app = require('../app.js');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/test-data');
+
+beforeEach(()=>{
+    return seed(data)
+})
+
+afterAll(()=>{
+    return db.end()
+})
+
+describe('NC news endpoint tests',()=>{
+    describe('GETting news',()=>{
+        test('GET /api/topics',()=>{
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body})=>{
+                expect(Array.isArray(body.topics)).toBe(true)
```

Comment:
You will need to assert the length of `body.topics` as if it is 0 the forEach will never call any of the expects. Also you know the length of the topics just by looking in the test data.